addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -205,6 +207,7 @@ def _compute_l10n_latam_document_type(self):
 
     @api.constrains('name', 'partner_id', 'company_id', 'posted_before')
     def _check_unique_vendor_number(self):
+        warnings.warn("Please install `l10n_latam_account_sequence`")
         """ The constraint _check_unique_sequence_number is valid for customer bills but not valid for us on vendor
         bills because the uniqueness must be per partner """
         for rec in self.filtered(
```

Comment:
```suggestion
    def _check_unique_vendor_number(self):
        """ The constraint _check_unique_sequence_number is valid for customer bills but not valid for us on vendor
        bills because the uniqueness must be per partner """
        warnings.warn("Please install `l10n_latam_account_sequence`")
        for rec in self.filtered(
```
the warning was inserted above the docstring.

`warnings.warning` also seems like the wrong API, it should be `logging.warning`. `warning` points to a specific location (not useful here), and has an entire traceback (for us).